.github/scripts/mqtt_test_v5.py

```diff
@@ -204,7 +204,7 @@ def test_topic_alias():
         time.sleep(1)
         times += 1
 
-    time.sleep(2)
+    time.sleep(5)
```

Comment:
5 is ok?

Reply:
It doesn't fundamentally solve the problem, but it can help alleviate it.